include/aspect/simulator.h

```diff
@@ -1148,6 +1152,15 @@ namespace aspect
       void compute_reactions ();
 
 
+      /**
+       * TODO: documentation.
+       *
+       * This function is implemented in
+       * <code>source/simulator/helper_functions.cc</code>.
+       */
+      void interpolate_material_output_into_field ();
```

Comment:
So you want to change that here too, using `interpolate_material_output_into_field(const AdvectionField &c)`.